### ISSUE TITLE
tweakers.net

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#4352]
### Basic Information

**Test link:** `https://tweakers.net/reviews/8004/community-interview-de-drumnbass-van-kaos001.html`
**Category:** ads
**Reported from:** United States
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.89 Safari/537.36
**Extension:** Nano Adblocker 1.0.0.152

### Additional Message

unblocked 1p tracking done by script `https://tweakers.net/x/scripts/min/1mdEg.js`

**Comment by llacb47** [comment]
Script URL changed to `https://tweakers.net/x/scripts/min/1XYXy.js`

**Comment by jspenguin2017** [comment]
Out of scope. Report to EasyPrivacy or uAssets. Marking as works.
